WordPress/gutenberg
# [Mobile] Add Cover block overlay color settings

### Related PR

`gutenberg-mobile`: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2242

## Description

This PR adds overlay color and gradient settings to the mobile Cover block. This resolves https://github.com/wordpress-mobile/gutenberg-mobile/issues/1735, and partially resolves https://github.com/wordpress-mobile/gutenberg-mobile/issues/1737.

## To test
1. In a new post, add a Cover block
2. Select an image from the media library
3. Tap the up-arrow on the floating toolbar (to make sure the Cover block is selected, not the inner paragraph block)
4. Tap the settings gear
5. Tap "Overlay color" 
6. Change the color in the following ways:
   1. Pick a pre-selected color from the palette
   2. Choose a custom color (scroll all the way to the right)
   3. Choose a pre-selected gradient (tap the gradient option on the bottom)

### Expected

Colors / gradients should update the block as they are selected.

## Screencast

<img src="https://user-images.githubusercontent.com/8507675/81381288-c9110080-914f-11ea-9d02-f3009202ea6d.gif" width="360">

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
